src/main/java/pl/isa/carservice/validators/datebeforeequalstoday/DateBeforeEqualsTodayValidator.java

```diff
@@ -0,0 +1,21 @@
+package pl.isa.carservice.validators.datebeforeequalstoday;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.time.LocalDate;
+
+public class DateBeforeEqualsTodayValidator implements ConstraintValidator<IsDateBeforeEqualsToday, LocalDate>{
+
+    @Override
+    public void initialize(IsDateBeforeEqualsToday date) {
+    }
+
+    @Override
+    public boolean isValid(LocalDate localDateField, ConstraintValidatorContext cxt) {
+        if (localDateField == null) {
+            return false;
+        }
+        LocalDate now = LocalDate.now();
+        return (localDateField.isBefore(now) || localDateField.isEqual(now));
```

Comment:
It can be simplified 😉 
```suggestion
        return localDateField != null && !localDateField.isAfter(LocalDate.now());
```